scripts/src/genesis_config_commands.sh

```diff
@@ -33,7 +33,7 @@ set_consenus_params() {
 	# 	MaxGas:   10000000, // ten million
     echo "setting consensus params"
     jq '.consensus_params["block"]["max_bytes"] = "4194304"' "$GENESIS_FILE" > "$tmp" && mv "$tmp" "$GENESIS_FILE"
-    jq '.consensus_params["block"]["max_gas"] = "10000000"' "$GENESIS_FILE" > "$tmp" && mv "$tmp" "$GENESIS_FILE"
+    jq '.consensus_params["block"]["max_gas"] = "-1"' "$GENESIS_FILE" > "$tmp" && mv "$tmp" "$GENESIS_FILE"
```

Comment:
why that? caused issues?

Reply:
not really it's just a common configuration we had in blubmus I believe at some point but can change it back.